docs/pages/workflow/debugging.md

```diff
@@ -96,9 +97,10 @@ We'll give a quick look at it here, but check out their [documentation](https://
 
 You can install it via the [release page](https://github.com/jhen0409/react-native-debugger/releases), or if you're on macOS you can run:
 
-```sh
-brew install --cask react-native-debugger
-```
+<Terminal
+  cmdCopy="brew install react-native-debugger"
+  cmd={['$ brew install react-native-debugger']}
```

Comment:
[Cask is dead, long live casks](https://github.com/Homebrew/homebrew-cask/blob/master/USAGE.md#how-to-use-homebrew-cask)